src/models/method.js

```diff
@@ -1,35 +1,35 @@
-import Model from 'model';
+import Model from "model";
 
 /**
  * The `Method` model
  */
 export default class Method extends Model {
-  static IDEAL = 'ideal';
-  static CREDITCARD = 'creditcard';
-  static BANCONTACT = 'bancontact';
-  static SOFORT = 'sofort';
-  static BANKTRANSFER = 'banktransfer';
-  static DIRECTDEBIT = 'directdebit';
-  static BITCOIN = 'bitcoin';
-  static PAYPAL = 'paypal';
-  static BELFIUS = 'belfius';
-  static PAYSAFECARD = 'paysafecard';
-  static PODIUMCADEAUKAART = 'podiumcadeaukaart';
-  static KBC = 'kbc';
-  static GIFTCARD = 'giftcard';
-  static INGHOMEPAY = 'inghomepay';
+  static IDEAL = "ideal";
+  static CREDITCARD = "creditcard";
+  static BANCONTACT = "bancontact";
+  static SOFORT = "sofort";
+  static BANKTRANSFER = "banktransfer";
+  static DIRECTDEBIT = "directdebit";
+  static BITCOIN = "bitcoin";
+  static PAYPAL = "paypal";
+  static BELFIUS = "belfius";
+  static PAYSAFECARD = "paysafecard";
+  static PODIUMCADEAUKAART = "podiumcadeaukaart";
+  static KBC = "kbc";
+  static GIFTCARD = "giftcard";
+  static INGHOMEPAY = "inghomepay";
 
   constructor(props) {
     super(props);
 
     const defaults = {
-      resource: 'method',
+      resource: "method",
```

Comment:
I see a lot of changes to use double quotes while you have configured single quotes in `.prettierrc`. We prefer single quotes in all our projects.

Reply:
Yeah, there's something wrong. I ran Prettier again this morning (just to be sure) and it doesn't seem to recognize the `.prettierrc` file. Will check it again later today 👍